sdk/pubkey/src/lib.rs

```diff
@@ -1,9 +1,142 @@
 #![no_std]
 
-pub use five8_const::decode_32_const;
-pub use pinocchio;
+#[cfg(feature = "const")]
+pub mod reexport {
+    pub use pinocchio::pubkey::Pubkey;
+}
```

Comment:
I think it is nice to add a `#[doc(hidden)]` attribute.
To avoid this macro being visible in the generated docs.

```suggestion
#[cfg(feature = "const")]
#[doc(hidden)]
// Re-export dependencies used in macros.
pub mod reexport {
    pub use pinocchio::pubkey::Pubkey;
}
```

Reply:
My bad, the intention as to have included the `#[doc(hidden)]` attribute.